install_config/install/disconnected_install.adoc

```diff
@@ -171,7 +171,20 @@ To sync the container images:
 # systemctl start docker
 ----
 
-. Pull all of the required {product-title} containerized components.
+. If you are performing a xref:rpm_vs_containerized.adoc[containerized install],
```

Comment:
xref needs an explicit anchor for our Customer Portal builds. Top of page for this would be `#install-config-install-rpm-vs-containerized`.